<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleRobotInfoWidget</class>
 <widget class="QWidget" name="SingleRobotInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="robotNameLabel">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Robot_0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Battery</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QProgressBar" name="batteryStatusBar">
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Connections</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QTableWidget" name="connectionTable">
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string>Robot</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Power</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Mission</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="missionLabel">
     <property name="text">
      <string>none</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="globalStopCheckBox">
     <property name="text">
      <string>Stop robot</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QCheckBox" name="explorationStopCheckBox">
     <property name="text">
      <string>Exploration</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QCheckBox" name="slamStopCheckBox">
     <property name="text">
      <string>Semantic Mapping</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="submitButton">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Modules activation:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QCheckBox" name="slamCheckBox">
     <property name="text">
      <string>SLAM</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QCheckBox" name="victimDetectionStopCheckBox">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Victim detection</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QPushButton" name="autoModeButton">
     <property name="text">
      <string>Autonomous Navigation</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QScrollBar" name="ContrastBar">
     <property name="maximum">
      <number>150</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QScrollBar" name="BrightnessBar">
     <property name="minimum">
      <number>-120</number>
     </property>
     <property name="maximum">
      <number>120</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Brightness</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Contrast</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QPushButton" name="victimFoundButton">
     <property name="text">
      <string>Victim Found</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QProgressBar" name="signalStrenghtBar">
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
